assets/sass/_cover.scss

```diff
@@ -56,7 +56,7 @@
 			margin-bottom: 0;
 		}
 
-		@media screen and (max-width: $width-tablet) {
+		@include max-width($width-tablet) {
```

Comment:
Expected item on line 59 to appear before line 50. Rule sets should be ordered as follows: `@extends`, `@includes` without `@content`, properties, `@includes` with `@content`, nested rule sets